src/main/java/org/dominokit/domino/ui/forms/TextFormElement.java

```diff
@@ -25,7 +25,16 @@ private void addFocusListeners() {
     public HTMLDivElement asElement() {
         return formGroup;
     }
+    
+	public boolean isDisabled() {
+		String disabled = getInputElement().getAttribute("disabled");
+		if(null == disabled) {
+			return false;
+		}
+		return true;
```

Comment:
This whole if/return section could be replaced with `return disabled != null;` (or `null != disabled` if you prefer).